<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exposure</class>
 <widget class="QWidget" name="Exposure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exposure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="auto_gb">
     <property name="title">
      <string>Auto</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Measured Brightness</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target Brightness</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bracket Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="AE_targetBrightness_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="AE_updateSpeed_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="AE_bracketSize_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Update Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="AE_enable_cb">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Auto (instant)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="AE_flickerRejectionHz_lb">
        <item>
         <property name="text">
          <string>50Hz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60Hz</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="AE_flickerRejectionEnable_cb">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Flicker Rejection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="AE_measuredBrightness_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="manual_gb">
     <property name="title">
      <string>Manual</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sensor Exposure</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Sensor Gain</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Exposure</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Gain</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="setGain_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="setExposure_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> us</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="sensorExposure_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> us</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="sensorGain_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AE_enable_cb</tabstop>
  <tabstop>AE_updateSpeed_sb</tabstop>
  <tabstop>AE_targetBrightness_sb</tabstop>
  <tabstop>AE_measuredBrightness_sb</tabstop>
  <tabstop>AE_bracketSize_sb</tabstop>
  <tabstop>AE_flickerRejectionEnable_cb</tabstop>
  <tabstop>AE_flickerRejectionHz_lb</tabstop>
  <tabstop>setExposure_sb</tabstop>
  <tabstop>sensorExposure_sb</tabstop>
  <tabstop>setGain_sb</tabstop>
  <tabstop>sensorGain_sb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
